CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 Change Log
 ==========
+2.0.0
+-----
+
+* Upgraded to Python 3.
+
```

Comment:
Could you also add lines to mention the changes reflected in these commits:
https://github.com/unt-libraries/aubreylib/commit/23a3979cc0c3a767b1f4ec6d4dee0c56aa4be020
https://github.com/unt-libraries/aubreylib/commit/264a6c17da4425d3a80998acca83792305592413

Reply:
will do.